test/NuGet.Core.FuncTests/NuGet.XPlat.FuncTest/XPlatLocalsTests.cs

```diff
@@ -47,53 +81,66 @@ public static void Locals_Succeeds(String args)
         [InlineData("locals -l")]
         [InlineData("locals --clear")]
         [InlineData("locals -c")]
-        public static void Locals_Success_InvalidArguments_HelpMessage(String args)
+        public static void Locals_Success_InvalidArguments_HelpMessage(string args)
         {
             // Arrange
-            var log = new TestCommandOutputLogger();
+            var expectedResult = string.Concat("error: usage: NuGet locals <all | http-cache | global-packages | temp> [--clear | -c | --list | -l]",
+                                               Environment.NewLine,
+                                               "error: For more information, visit http://docs.nuget.org/docs/reference/command-line-reference");
```

Comment:
you should have specific messages about what is missing from the `args` and assertions on those